iphone/Maps/UI/Help/AboutController/AboutController.swift

```diff
@@ -0,0 +1,535 @@
+    let data = infoContent.map { [weak self] aboutInfo in
+      return AboutInfoTableViewCellModel(title: aboutInfo.title, image: aboutInfo.image, didTapHandler: {
+        switch aboutInfo {
+        case .faq:
+          self?.navigationController?.pushViewController(FaqController(), animated: true)
+        case .reportABug:
+          guard let link = aboutInfo.link else { return }
+          self?.sendEmailWith(header: "Organic Maps Bugreport", toRecipients: [link])
+        case .reportMapDataProblem, .volunteer, .news:
+          self?.openUrl(aboutInfo.link)
+        case .rateTheApp:
+          UIApplication.shared.rateApp()
+        default:
+          break
+        }
+      })
+    }
+    return data
+  }
+
+  func buildSocialMediaCollectionViewData() -> [SocialMediaCollectionViewCellModel] {
+    let socialMediaContent: [SocialMedia] = [.telegram, .github, .instagram, .twitter, .linkedin, .organicMapsEmail, .reddit, .matrix, .facebook, .fosstodon]
+    let data = socialMediaContent.map { [weak self] socialMedia in
+      return SocialMediaCollectionViewCellModel(image: socialMedia.image, didTapHandler: {
+        switch socialMedia {
+        case .telegram: fallthrough
+        case .github: fallthrough
+        case .reddit: fallthrough
+        case .matrix: fallthrough
+        case .fosstodon: fallthrough
+        case .facebook: fallthrough
+        case .twitter: fallthrough
+        case .instagram: fallthrough
+        case .linkedin:
+          self?.openUrl(socialMedia.link, inSafari: true)
+        case .organicMapsEmail:
+          guard let link = socialMedia.link else { return }
+          self?.sendEmailWith(header: "Organic Maps", toRecipients: [link])
+        }
+      })
+    }
+    return data
+  }
+
+  // Returns a human-readable maps data version.
+  static func formattedMapsDataVersion() -> String {
+    // First, convert version code like 220131 to a date.
+    let df = DateFormatter()
+    df.locale = Locale(identifier:"en_US_POSIX")
+    df.dateFormat = "yyMMdd"
+    let mapsVersionInt = FrameworkHelper.dataVersion()
+    let mapsDate = df.date(from: String(mapsVersionInt))!
+    // Second, print the date in the local user's format.
+    df.locale = Locale.current
+    df.dateStyle = .long
+    df.timeStyle = .none
+    return df.string(from:mapsDate)
+  }
+
+  static func formattedAppVersion() -> String {
+    let appInfo = AppInfo.shared();
+    // Use strong left-to-right unicode direction characters for the app version.
+    return String(format: L("version"), "\u{2066}\(appInfo.bundleVersion)-\(appInfo.buildNumber)\u{2069}")
+  }
+
+  func showCopyright() {
+    let path = Bundle.main.path(forResource: "copyright", ofType: "html")!
+    let html = try! String(contentsOfFile: path, encoding: String.Encoding.utf8)
+    let webViewController = WebViewController.init(html: html, baseUrl: nil, title: L("copyright"))!
```

Comment:
Does it properly work with the dark theme?

Reply:
Yes!
<img width="350" alt="image" src="https://github.com/organicmaps/organicmaps/assets/79797627/8e8568dd-f09d-4261-a141-c956fb10d7d8">